src/widget/woverview.cpp

```diff
@@ -338,6 +338,8 @@ void WOverview::paintEvent(QPaintEvent * /*unused*/) {
             }
 
             painter.drawImage(rect(), m_waveformImageScaled);
+            // desaturate the scaled waveform-image up to the current play-position by overdrawing semi-transparent black
```

Comment:
Since the color is now customizable the comment is outdated. 
Please skip this line if the color is fully transparent.